hibernate-core/src/main/java/org/hibernate/engine/jdbc/dialect/internal/StandardDialectResolver.java

```diff
@@ -187,7 +187,6 @@ else if ( majorVersion == 10 && minorVersion == 5 ) {
 					}
 					else {
 						// assume `majorVersion > 13`
-						LOG.unknownSqlServerVersion( majorVersion, SQLServer2012Dialect.class );
```

Comment:
hm yeah, would have added to case instead, but I guess this is less maintenance.

Reply:
Took it from master where it is like this.

Reply:
The switch itself didn't change anyway, just the log message has been removed, or?

Reply:
yes, but including case 15 in the switch wqould not log the default warning for 2019
but the outcome for 2019 is the same - no warning and using 2012 dialect.

Reply:
hm, I have actually approved the change, no idea why it dit not show up as approved.